code/src/main/java/com/onlinecontacttracing/servlets/authentication/CheckForApiAuthorizationServlet.java

```diff
@@ -35,12 +35,12 @@
 @WebServlet("/check-for-api-authorization")
```

Comment:
We should probably remove this line - since this is an abstract class, it's not instantiable, and would be able to serve a request at this (or any other) URI.

Reply:
done